### PR TITLE
zimpl 3.5.3 (new formula)

### PR DESCRIPTION
**Pull request by jurgen-lentz**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/zimpl.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+class Zimpl < Formula
+  desc "Language to translate the mathematical model of a problem into a MI(N)LP"
+  homepage "https://zimpl.zib.de"
+  url "https://scipopt.org/download/release/scipoptsuite-8.0.3.tgz"
+  version "3.5.3"
+  sha256 "5ad50eb42254c825d96f5747d8f3568dcbff0284dfbd1a727910c5a7c2899091"
+  license "LGPL-3.0-only"
+
+  depends_on "cmake" => :build
+  depends_on "bison"
```
Comment (SMillerDev) [inline Formula/zimpl.rb:10]: can't use the one from macOS?

File: Formula/zimpl.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
+class Zimpl < Formula
+  desc "Language to translate the mathematical model of a problem into a MI(N)LP"
+  homepage "https://zimpl.zib.de"
+  url "https://scipopt.org/download/release/scipoptsuite-8.0.3.tgz"
+  version "3.5.3"
```
Comment (SMillerDev) [inline Formula/zimpl.rb:5]: Is there no direct download?

File: Formula/zimpl.rb (hunk 1 above)
Comment (jurgen-lentz) [reply Formula/zimpl.rb:10]: Unfortunately, not. Bison at least 2.4.0 is required.

File: Formula/zimpl.rb (hunk 2 above)
Comment (jurgen-lentz) [reply Formula/zimpl.rb:5]: Unfortunately, not. Only old versions are downloadable through https://zimpl.zib.de.

**Comment by jurgen-lentz** [comment]
Can be merged.

**Comment by carlocab** [comment]
Can we just make this part of the `scip` formula in #124261?

**Comment by jurgen-lentz** [comment]
> Can we just make this part of the `scip` formula in #124261?

```zimpl``` can also be used by ```soplex``` or ```lp_solve``` for example. I think it is better to leave it separate.

File: Formula/zimpl.rb (hunk 2 above)
Comment (SMillerDev) [reply Formula/zimpl.rb:5]:
```
zimpl: livecheck error
Unable to get versions
```

**Comment by jurgen-lentz** [comment]
I think we should wait until ZIMPL is available on Github.

**Pull request closed without merging**